### PR TITLE
Added ipinfo.io support

### PR DESCRIPTION
**Pull request by Gurkengewuerz**
Added the API from [ipinfo.io](http://ipinfo.io/). To make it work i had to add jq to decode the json response. If someone has a better option please say it.

**Example:**
```
root@MVS1936:~# climate ipinfo
+ curl --silent https://ipinfo.io/
IP-Address: 130.180.88.76
Hostname: b2b-130-180-88-76.unitymedia.biz
Network: AS6830 Liberty Global Operations B.V.
City: Dortmund, North Rhine-Westphalia, DE
Postal Code: 44143
Latitude/Longitude: 51.5208,7.5184

root@MVS1936:~# climate ipinfo 198.41.214.162
+ curl --silent https://ipinfo.io/198.41.214.162
IP-Address: 198.41.214.162
Hostname: No Hostname
Network: AS13335 Cloudflare, Inc.
City: San Francisco, California, US
Postal Code: 94107
Latitude/Longitude: 37.7697,-122.3933
```

### CODE REVIEW

**Comment by Gurkengewuerz** [comment]
@adtac noticed the PR?

File: climate [hunk 1]
```diff
@@ -573,6 +602,9 @@ elif [[ $command == "is-up" ]]; then
         is-up "$commandargs"
     fi
 
+elif [[ $command == "ipinfo" ]]; then
+    ipinfo "$2"
```
Comment (adtac) [inline climate]: Not `"$2"`. Use `"$commandargs"` - see some of the other commands above.

**Comment by adtac** [comment]
Hey @Gurkengewuerz 

Sorry about that, I must have missed the notification somehow. Anyway, I left a tiny review comment. If that's patched up, we're good to go :)

**Comment by Gurkengewuerz** [comment]
@adtac done ;)

**Comment by adtac** [comment]
Cheers, thanks! Merged it :+1:

**Pull request merged**